components/membership/registration-impl/src/integrationTest/kotlin/net/corda/membership/impl/registration/MemberRegistrationIntegrationTest.kt

```diff
@@ -294,10 +294,10 @@ class MemberRegistrationIntegrationTest {
 
     @Test
     fun `dynamic member registration service publishes unauthenticated message to be sent to the MGM`() {
-        groupPolicyProvider.putGroupPolicy(TestGroupPolicy())
-
+        val groupPolicy = TestGroupPolicy()
         val member = HoldingIdentity(memberName, groupId)
         val context = buildTestContext(member)
+        groupPolicyProvider.putGroupPolicy(member, groupPolicy)
```

Comment:
You can just do this, since groupPolicy is a one-time used val:
```suggestion
        groupPolicyProvider.putGroupPolicy(member, TestGroupPolicy())
```